app/schema/widgets/radio_group_widget.py

```diff
@@ -0,0 +1,47 @@
+from app.schema.widget import Widget
+from flask import render_template
+from app.libs.utils import ObjectFromDict
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class RadioGroupWidget(Widget):
+    def __init__(self, name):
+        super().__init__(name)
+
+    def get_user_input(self, post_vars):
```

Comment:
Across a couple of the widgets this seems boilerplate - can we move it to the base class to make this DRY'er?